src/Raven.Studio/wwwroot/App/views/database/indexes/deleteIndexesConfirm.html

```diff
@@ -7,11 +7,26 @@
             <h4 class="modal-title" data-bind="text: title"></h4>
         </div>
         <div class="modal-body">
-            <p data-bind="visible: indexNames.length === 1">You're deleting <strong data-bind="text: indexNames[0]"></strong></p>
-            <p data-bind="visible: indexNames.length > 1">You're deleting <strong data-bind="text: indexNames.length"></strong> indexes</p>
-            <ul data-bind="visible: indexNames.length > 1, foreach: indexNames" style="max-height: 200px; overflow-y: auto;">
-                <li data-bind="text: $data"></li>
+            <p data-bind="html: subTitleHtml"></p>
+            <ul data-bind="foreach: indexesInfoForDelete()" style="max-height: 200px; overflow-y: auto;">
+                <li class="padding padding-xs">
+                    <strong data-bind="text: $data.indexName"></strong>
+                    <small data-bind="if: $data.reduceOutputCollection">
+                         <!-- The span code below is formatted like this becuase otherwise we get extra spaces in ui -->
+                        <span class="margin-left margin-left-sm">(</span><span 
+                            data-bind="text: $data.ReferenceCollection ? 'Reduce Results Collections:' : 'Reduce Results Collection:'"></span>
+                            <span data-bind="text: $data.reduceOutputCollection"></span><span data-bind="if: $data.ReferenceCollection"><span 
+                            data-bind="text: ', ' + $data.ReferenceCollection"></span></span><span>)</span>
+                    </small>
+                </li>
             </ul>
+            <div data-bind="if: warning" class="flex-horizontal text-warning bg-warning margin margin-top-lg">
+                <small class="flex-start padding padding-xs">
+                    <i class="icon-warning"></i>
+                </small>
+                <small data-bind="html: warning" class="padding padding-xs">
```

Comment:
I wonder if we should use small here - it is warning something should pay attention, on the other hand it is small to avoid paying attention ;)

Reply:
It looks ok with small and with 'regular' size
Switching to regular - so it looks like this:

![image](https://user-images.githubusercontent.com/17376974/70860221-9dcf1000-1f27-11ea-9364-c442cf705f01.png)

Reply:
let's leave regular  - we had few reports that font size is too small it is better to use normal font size especially for warnings.

Reply:
done (regular)